.github/workflows/ci.yml

```diff
@@ -8,166 +8,160 @@ on:
     branches-ignore: [staging.tmp]
 
 jobs:
-  ios_build:
-    name: iOS Stable
-    runs-on: macos-10.15
-    env:
-      TARGET: aarch64-apple-ios
-    steps:
-      - uses: actions/checkout@v2
-      - run: rustup component add clippy
-      - run: rustup target add ${{ env.TARGET }}
-      - run: cargo clippy --target ${{ env.TARGET }}
-
-  android_build:
-    name: Android Stable
-    runs-on: ubuntu-latest
-    env:
-      TARGET: aarch64-linux-android
-      PKG_CONFIG_ALLOW_CROSS: 1
-    steps:
-      - uses: actions/checkout@v2
-      - run: echo "$ANDROID_HOME/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin" >> $GITHUB_PATH
-      - run: rustup component add clippy
-      - run: rustup target add ${{ env.TARGET }}
-      - run: cargo clippy --target ${{ env.TARGET }}
-      - name: Additional core features
-        run: cargo check --manifest-path wgpu-core/Cargo.toml --features trace --target ${{ env.TARGET }}
-
-  wasm:
-    name: Web Assembly
-    runs-on: ubuntu-latest
-    env:
-      RUSTFLAGS: --cfg=web_sys_unstable_apis
-    steps:
-    - uses: actions/checkout@v2
-    - run: rustup target add wasm32-unknown-unknown
-    - name: Check WebGPU
-      run: cargo check --package wgpu --examples --target=wasm32-unknown-unknown
-    # disable until hal/Gles backend is setup
-    # - name: Check WebGL
-    #  run: cargo check --all-targets --target=wasm32-unknown-unknown --features webgl
-
   build:
-    name: ${{ matrix.name }}
-    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-10.15, ubuntu-20.04, windows-2019]
-        channel: [stable, nightly]
         include:
-          - name: MacOS Stable
+          # Windows
+          - name: Windows x86_64
+            os: windows-2019
             channel: stable
+            target: x86_64-pc-windows-msvc
+            kind: compile
+          
+          # MacOS
+
+          # Mac has no software runners, so don't run tests
+          - name: MacOS x86_64
             os: macos-10.15
-            prepare_command:
-            clippy_params:
-            additional_core_features:
-            additional_player_features: winit
-            wrapper_features: trace
-          - name: MacOS Nightly
+            channel: stable
+            target: x86_64-apple-darwin
+            kind: compile
+
+          
+          # IOS
+          - name: IOS Nightly x86_64
             os: macos-10.15
             channel: nightly
```

Comment:
We'd probably want to include iOS stable instead of nightly unless we can't do that for some reason

Reply:
Ack, yeah this should be stable. The old ci script had nightly meaning not running tests, so I forgot to change it back to stable.